pom.xml

```diff
@@ -51,7 +52,17 @@
         <maven.compiler.source>1.8</maven.compiler.source>
         <maven.compiler.target>1.8</maven.compiler.target>
         <junit-jupiter.version>5.5.2</junit-jupiter.version>
+        <guava.version>28.1-jre</guava.version>
     </properties>
+    <dependencies>
+        <dependency>
+            <!-- @VisibleForTesting Annotation -->
+            <groupId>com.google.guava</groupId>
+            <artifactId>guava</artifactId>
+            <version>${guava.version}</version>
+            <scope>test</scope>
```

Comment:
`Annotates a program element that exists, or is more widely visible than otherwise necessary, only for use in test code.`

Since it's purpose is to annotate production code I'd completely remove the dependency here, since it's only scoped to `test` anyway.